semanticdb/semanticdb3/javacp.md

```diff
@@ -445,13 +446,54 @@ Notes:
   parameters matches their name written in source. Otherwise, parameters have
   the name `paramN` where `N` is the index of that given parameter starting at
   index 0.
+* Supported accessibilities for `METHOD` symbols are
+  * `PRIVATE`: set for `private` methods
+  * `PRIVATE_WITHIN`: set for methods with package access
+  * `PUBLIC`: set for `public` methods
```

Comment:
What about `protected`? I propose a formulation similar to the one from the Scala part of the spec: "Method symbols support [all Scala accessibilities](#scala-accessibility)".